Issue #1323 - Add support for ':of-type' modifier (#2116)
IBM/FHIR: docs/src/pages/Conformance.md

Due to performance implications, the `:exact` modifier should be used for String searches where possible.

> The `:text`, `:above`, `:below`, `:in`, and `:not-in` modifiers are not supported in this version of the IBM FHIR server and use of these modifiers will result in an HTTP 400 error with an OperationOutcome that describes the failure.

Reviewer: Totally unrelated to your change, but I just noticed that we say `:above` and `:below` aren't supported, but we do actually support those for URI types...we just don't support those on token types.  Maybe we can reword this to be more clear?
```suggestion
The `:text`, `:above`, `:below`, `:in`, and `:not-in` token modifiers are not yet supported by the IBM FHIR server and use of these modifiers will result in an HTTP 400 error with an OperationOutcome that describes the failure.
```

The `:text` modifier actually applies to String types too, and we don't support it there either, but I think thats just too much information to distinguish...they can infer it from the table above.

Author: Reworded.
